lib/elixir_analyzer/exercise_test/common_checks.ex

```diff
@@ -47,6 +48,7 @@ defmodule ElixirAnalyzer.ExerciseTest.CommonChecks do
       ExemplarComparison.run(code_ast, type, exemploid_ast),
       Indentation.run(code_ast, code_string),
       PrivateHelperFunctions.run(code_ast, exemploid_ast),
+      FunctionCapture.run(code_ast),
```

Comment:
It would be great if there was at least one test somewhere that is going to fail if we accidentally remove this line, probably in `common_checks_test.exs`. I noticed that's not the case here, and also for `BooleanFunctions.run(code_ast)` 🙁

Reply:
Done!